setup.py

```diff
@@ -41,7 +41,7 @@
 skbio_3 = "scikit-bio >= 0.4.1"
 base = ["numpy >= 1.7", "scipy >= 0.17.0", "click", "pandas",
         skbio_2, "jinja2 >= 2.9", "future"]
-doc = ["Sphinx < 1.6", "sphinx-bootstrap-theme"]
+doc = ["Sphinx", "sphinx-bootstrap-theme"]
```

Comment:
We did not have a super smooth transition from Sphinx 1 -> 2 in QIIME 2, hope this goes better for you...

Reply:
Thanks for the heads up, after you posted this comment I looked again and realized that the travis file and setup.py files were out of sync. I fixed that and errors ensued. For now we'll keep it at <1.6. ⏳ 

This PR now only installs requests using conda, which should solve the broken build we found in #724.